docs/proposals/node-agent/README.md

```diff
@@ -0,0 +1,41 @@
+# Node Agent
+
+## Requirements
+
+### Which service must the domain provide
+
+The node agent amanges workload management. A workload is the tiniest unit in orka. It can be a container, a virtual machine, etc.
+
+### What is in scope and out of scope
+
+#### In scope
+
+The Node Agent provides an API :
+
+- to control the workloads :
+  - start
+  - stop
+  - restart
+- health of node
+- metrics of nodes
+- status of workloads
+  - waiting
+  - running
+  - terminated
```

Comment:
This describes that `Status` should be called by the Scheduler. Instead, the docs (namely slide n°79) stipulates that:

> Agent sends nodes and instances updates.

Please review the Scheduler's `agent.proto` implementation so we can plan together.

Reply:
That's correct. The proposed architecture has the agent sending nodes and instances updates to the scheduler.